packages/block-library/src/template-part/edit/category-panel.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { PanelBody, SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export default function TemplatePartCategoryPanel( { postId } ) {
+	const [ area, setArea ] = useEntityProp(
+		'postType',
+		'wp_template_part',
+		'area',
+		postId
+	);
+
+	return (
+		<PanelBody title={ __( 'Category' ) }>
```

Comment:
I'm not sure this should be in its own panel and with this title. Same remark about the below label `Select template part category`. I think we want to avoid `template part` wording and maybe just `Area` should be okay? 🤔 
@mtias

Reply:
For now, I followed @jameskoster 's mockup that you can find here: https://github.com/WordPress/gutenberg/issues/27875